transformation/src/test/scala/org/broadinstitute/monster/hca/HcaPipelineBuilderSpec.scala

```diff
@@ -290,4 +286,47 @@ class HcaPipelineBuilderSpec
       HcaPipelineBuilder.validateJson(sc.parallelize(Seq(exampleUrlAndFile)))
     } should have message "java.lang.Exception: Data does not conform to schema: NonEmptyList(#: required key [schema_type] not found)"
   }
+
+  it should "not mutate the json when validating" in {
+    val exampleFileContent = JsonParser.parseEncodedJson(
+      """
+        |{
+        |    "organ": {
+        |        "text": "brain",
+        |        "ontology": "astrocyte"
+        |    },
+        |    "schema_type": "biomaterial",
+        |    "biomaterial_core": {
+        |        "ncbi_taxon_id": [
+        |            9606
+        |        ],
+        |        "biomaterial_id": "Q4_DEMO-sample_SAMN02797092",
+        |        "has_input_biomaterial": "Q4_DEMO-donor_MGH30",
+        |        "biomaterial_name": "Q4_DEMO-Single cell mRNA-seq_MGH30_A01",
+        |        "supplementary_files": [
+        |            "Q4_DEMO-protocol"
+        |        ]
+        |    },
+        |    "organ_part": {
+        |        "text": "glioblastoma"
+        |    },
+        |    "genus_species": [
+        |        {
+        |            "text": "Homo sapiens",
+        |            "ontology": "NCBITaxon:9606"
+        |        }
+        |    ],
+        |    "describedBy": "https://schema.humancellatlas.org/type/biomaterial/5.1.0/specimen_from_organism"
+        |}
+        |""".stripMargin
+    )
+
+    val exampleUrlAndFile = (exampleFileContent.read[String]("describedBy"), exampleFileContent)
```

Comment:
Should this be a file path instead of a URL? I think we talked about the same thing [here](https://github.com/DataBiosphere/hca-ingest/pull/25/files#r465936121), but it might have been copied into this test too.

Reply:
Good catch, I incorrectly copied and pasted.